lib/career_bot/responders/message_responder.rb

```diff
@@ -3,32 +3,44 @@ def initialize(message)
     @message = message
   end
 
-  def set_action
-    case
-      when message.text == 'test'
-        message.reply(I18n.t('test_message'))
-      when message.attachments && conversation.apply == false
-        message.reply(I18n.t('attachment_response'))
-        message.reply(gif_response)
-      when conversation.apply
-        run_apply_process
-      when quick_reply && quick_reply != 'empty'
-        run_postback
-      else
-        send_to_wit
-    end
+  def response
+    test_message
```

Comment:
Why can't we use case statement here instead of separating `test_message` and `set_action` with elseifs? We could handle test message there

Reply:
👍